src/mbgl/map/map.cpp

```diff
@@ -1095,11 +1099,11 @@ void Map::Impl::onUpdate(Update flags) {
 }
 
 void Map::Impl::onStyleLoaded() {
-    backend.notifyMapChange(MapChangeDidFinishLoadingStyle);
+    observer.onDidFinishLoadingStyle();
 }
 
 void Map::Impl::onStyleError() {
-    backend.notifyMapChange(MapChangeDidFailLoadingMap);
+    observer.onDidFailLoadingMap();
```

Comment:
This suggestion isn’t platform-specific but rather an improvement on the code being added in this PR. Tracking in #8398.